alerting/build.gradle

```diff
@@ -29,6 +30,17 @@ opensearchplugin {
     extendedPlugins = ['lang-painless']
 }
 
+publishing {
+    publications {
+        pluginZip(MavenPublication) { publication ->
+            pom {
+                name = "opensearch-alerting"
+                description = "OpenSearch Alerting plugin"
+            }
```

Comment:
If not please also add `licenses` and `developers` info.

Reply:
Added